src/pages/workspace/tags/WorkspaceTagsPage.tsx

```diff
@@ -350,7 +350,7 @@ function WorkspaceTagsPage({route}: WorkspaceTagsPageProps) {
                         customListHeader={getCustomListHeader()}
                         shouldPreventDefaultFocusOnSelectRow={!DeviceCapabilities.canUseTouchScreen()}
                         listHeaderWrapperStyle={[styles.ph9, styles.pv3, styles.pb5]}
-                        onDismissError={(item) => Tag.clearPolicyTagErrors(policyID, item.value)}
+                        onDismissError={(item) => Tag.clearPolicyTagErrors(policyID, item.value, item.orderWeight ?? 0)}
```

Comment:
```suggestion
                        onDismissError={(item) => !isMultiLevelTags && Tag.clearPolicyTagErrors(policyID, item.value, 0)}
```

In this list we may render either tags or taglists.
- If we render tags (!isMultiLevelTags), we call clearPolicyTagErrors and the tag list index is 0
- If we render tag lists, let's do nothing for now as I don't see a clear case yet